app/views/claims/support/claims/index.html.erb

```diff
@@ -1,5 +1,38 @@
 <%= render "claims/support/primary_navigation", current: :claims %>
+<%= content_for :page_title, t(".heading") %>
 
 <div class="govuk-width-container">
-  <h1 class="govuk-heading-l"><%= t(".heading") %></h1>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-l"><%= t(".heading") %></h1>
+
+    <% if @claims.any? %>
+      <table class="govuk-table">
+        <thead class="govuk-table__head">
+          <tr class="govuk-table__row">
+            <th scope="col" class="govuk-table__header"><%= t(".id") %></th>
+            <th scope="col" class="govuk-table__header"><%= t(".state") %></th>
+          </tr>
+        </thead>
+
+        <tbody class="govuk-table__body">
+          <% @claims.each do |claim| %>
+            <tr class="govuk-table__row">
+              <td class="govuk-table__cell"><%= govuk_link_to claim.id, claims_support_claim_path(claim) %></td>
+              <td class="govuk-table__cell"><%= render(Claim::StatusTagComponent.new(claim:)) %></td>
+            </tr>
+          <% end %>
+        </tbody>
+      </table>
```

Comment:
Let's use the [govuk_table](https://govuk-components.netlify.app/components/table/) helper/component here.

Reply:
Fixed https://github.com/DFE-Digital/itt-mentor-services/pull/220/commits/e67a5723e9988f8d55656d016a69c555972213c4